recipes/phyde/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "phyde" %}
+{% set version = "0.4.0" %}
+{% set sha256 = "f7ed67c8a77dd40f8009fb87334f35dcc68fdf5524b8121223cbaef3798667b3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [win or (not (py27 or py36))]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - toolchain
+    - python
+    - setuptools
+    - subprocess32  # [not py36]
+    - cython
+    - pandas
+    - numpy
+    - matplotlib
+    - seaborn
+    - multiprocess
+
+  run:
+    - python
+    - subprocess32  # [not py36]
+    - cython
+    - pandas
+    - numpy
+    - matplotlib
+    - seaborn
+    - multiprocess
+
+test:
+  imports:
+    - phyde
```

Comment:
can you please include some more import tests?

Reply:
I just added some more import tests. Please let me know if you want me to add any others.

Reply:
That looks good!